components/common/btn/followButtons.tsx

```diff
@@ -1,30 +1,28 @@
 import styled from '@emotion/styled';
-
-interface FollowBtnProps {
-  bgColor: string;
+interface InFollowButtonpropsTypes {
+  backgroundColor: string;
   children: React.ReactNode;
 }
 
-const Follow = styled.button<FollowBtnProps>`
+export default function FollowButtons({ children, backgroundColor }: InFollowButtonpropsTypes) {
+  return <Follow backgroundColor={backgroundColor}>{children}</Follow>;
+}
+
+const Follow = styled.button<InFollowButtonpropsTypes>`
   display: flex;
   align-items: center;
   justify-content: center;
 
   height: 2.5rem;
   padding: 0.7rem 0.9rem;
 
-  font-family: 'SF_HailSnow';
-  font-size: 1rem;
-  color: white;
+  color: ${({ theme }) => theme.color.white};
 
-  background-color: ${props => props.bgColor};
-  border: 1px solid var(--black, #000);
-  border-radius: 1.5rem;
-  box-shadow: 0rem 0.4rem 0rem 0.4rem rgba(0, 0, 0, 0.25);
-`;
+  background-color: ${props => props.backgroundColor};
```

Comment:
```suggestion
  background-color: ${({ theme, bgColor }) => theme.color[bgColor]};
```
요러면 되긴 하는데 그 theme type에 bgColor가 없다고 뜨긴 해..........허허